src/transport_performance/gtfs/report/report_utils.py

```diff
@@ -0,0 +1,183 @@
+"""Utils to assist in the creation of a HTML report for GTFS."""
```

Comment:
I expect the methods for this class to be internal only. I would have thought users would want the reports. But we can relax the testing for internals. Calling things like `TemplateHTML.insert()` would be for development only, right?